src/main/kotlin/racing/Racing.kt

```diff
@@ -0,0 +1,25 @@
+package racing
+
+object Racing {
+    fun start(carNumber: Int, tryRound: Int): ArrayList<Array<Int>> {
```

Comment:
리턴값은 구현체(ArrayList)가 아닌 인터페이스를 리턴해주면 어떨까요?

```suggestion
    fun start(carNumber: Int, tryRound: Int): List<Array<Int>> {
```